---
omronsentech_camera/DeviceConnection[] device_list   # list of detected device
